src/partners/paybis.ts

```diff
@@ -289,3 +299,45 @@ export function processPaybisTx(rawTx: unknown): StandardTx {
   }
   return standardTx
 }
+
+function getFiatPaymentType(
+  tx: PaybisTx,
+  direction: 'buy' | 'sell'
+): FiatPaymentType | null {
+  const name = direction === 'buy' ? tx.from.name : tx.to.name
+  switch (name) {
+    case undefined:
+      return null
+    case 'AstroPay':
+      // Idk?
+      return null
+    case 'Credit/Debit Card':
+      return 'credit'
+    case 'FPX':
```

Comment:
did someone actually use FPX from Edge? IF so, just add `fpx` as a payment type

Reply:
I'll double-check. And then I'll verify what that is with Paybis.

Reply:
FPX exists. Where can I reach out to Paybis?